app/src/main/java/com/example/securityapplication/MainActivity.java

```diff
@@ -122,10 +183,9 @@ public void onComplete(@NonNull Task<AuthResult> task) {
 
     private void signOut(){
         mAuth.signOut();
-        updateUI(null);
     }
 
-    private void updateUI(FirebaseUser user){
+    public void updateUI(FirebaseUser user){
```

Comment:
@Rohan-RPJ FaceBookButton and GoogleSignIn button were to be disabled if logged in successfully.